storage/client/src/samples/java/blob/FileTransferExample.java

```diff
@@ -0,0 +1,171 @@
+
+package blob;
+
+import com.azure.core.util.configuration.ConfigurationManager;
+import com.azure.storage.blob.BlockBlobClient;
+import com.azure.storage.blob.ContainerClient;
+import com.azure.storage.blob.StorageClient;
+import com.azure.storage.common.credentials.SharedKeyCredential;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.RandomAccessFile;
+import java.net.URL;
+import java.nio.file.Files;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.util.Locale;
+
+public class FileTransferExample {
+    private static final String LARGE_TEST_FOLDER = "test-large-files/";
+    public static void main(String[] args) {
+        // From the Azure portal, get your Storage account's name and account key.
+        String accountName = getAccountName();
+        String accountKey = getAccountKey();
+        // Use your Storage account's name and key to create a credential object; this is used to access your account.
+        SharedKeyCredential credential = new SharedKeyCredential(accountName, accountKey);
+
+        /**
+         * From the Azure portal, get your Storage account blob service URL endpoint.
+         * The URL typically looks like this:
+         */
+        String endPoint = String.format(Locale.ROOT, "https://%s.blob.core.windows.net", accountName);
+
+        // Create a StorageClient object that wraps the service endpoint, credential and a request pipeline.
+        // Now you can use the storageClient to perform various container and blob operations.
+        StorageClient storageClient = StorageClient.storageClientBuilder().endpoint(endPoint).credential(credential).buildClient();
+
+
+        // This example shows several common operations just to get you started.
+
+        /*
+        Create a client that references a to-be-created container in your Azure Storage account. This returns a
+        ContainerClient object that wraps the container's endpoint, credential and a request pipeline (inherited from storageClient).
+        Note that container names require lowercase.
+         */
+        ContainerClient containerClient = storageClient.getContainerClient("myjavacontainerparallelupload" + System.currentTimeMillis());
+
+        // Create a container in Storage blob account.
+        containerClient.create();
+        /*
+        Create a client that references a to-be-created blob in your Azure Storage account's container.
+        This returns a BlockBlobClient object that wraps the blob's endpoint, credential and a request pipeline
+        (inherited from containerClient). Note that blob names can be mixed case.
+         */
+        // Create a BlockBlobClient object that wraps a blob's endpoint and a default pipeline.
+        String filename = "BigFile.bin";
+        BlockBlobClient blobClient = containerClient.getBlockBlobClient(filename);
```

Comment:
I don't think the "(inherited from containerClient)" is correct. And can you add a comment that calls out BlockBlobClient is what gives us access to upload?

Reply:
I have checked the code. It is true that the blob client inherited endpoint, credential and a request pipeline from storageClient. I will leave the comments over there.

I will add another comment to call out we use blockblob to upload.